app/spdb/build.sbt

```diff
@@ -233,16 +233,16 @@ def fnussber(version: Version) = AppConfig(
     "-XX:MaxPermSize=196M",
     "-Dedu.gemini.site=north",
     "-Dcron.*.edu.gemini.dbTools.html.ftpHost=localhost",
-    "-Dcron.*.edu.gemini.dbTools.html.ftpDestDir=/Users/fnussber/.spdb/sftp",
+    "-Dcron.*.edu.gemini.dbTools.html.ftpDestDir=/Users/osmirnov/.spdb/sftp",
```

Comment:
I deleted florian's spdb profile, It is better if you add your own profile with your name (osmirnov). Remember to add it to the list on line 25. If this is complicated I can do it for you when I merge my  PR #941